anthos-onprem-terraform/avmw_user_cluster_metallb/README.md

```diff
@@ -50,3 +50,29 @@ have changed directory to where this samples is:
 
     You can view your user cluster in the
     [Anthos clusters page](https://console.cloud.google.com/anthos/clusters).
+
+### Upgrade the user cluster with terraform
+
+Before upgrading the user cluster, please make sure the platform controller of the target version has been upgraded. The command to upgrade the platform controller is
+
+```
+gkectl prepare --upgrade-platform
```

Comment:
The full command that we need in docs is:

gkectl prepare \
    --bundle-path /var/lib/gke/bundles/gke-onprem-vsphere-TARGET_VERSION.tgz \
    --kubeconfig ADMIN_CLUSTER_KUBECONFIG \
    --upgrade-platform

Reply:
Corrected. Thanks for the guidance